openapi/openapi.yaml

```diff
@@ -463,7 +472,7 @@ paths:
                 type: array
                 items:
                   $ref: "#/components/schemas/ToolFile"
-            text/plain:
+            application/zip:
```

Comment:
Did you remove `text/plain` on purpose?

Reply:
Yes, as a side effect.
```
produces:
  - application/json
  - text/plain
```
is mentioned at the root of the yaml which when converted to openapi.yaml will put create two responses for every endpoint (one for each type) which is not quite correct for most endpoints.  I've overwritten this endpoint to be application/json and application/zip.